docs/reference/query-dsl/fuzzy-query.asciidoc

```diff
@@ -1,75 +1,99 @@
 [[query-dsl-fuzzy-query]]
 === Fuzzy Query
 
-The fuzzy query uses similarity based on Levenshtein edit distance.
+Returns documents that contain terms similar to the search term.
 
-==== String fields
+{es} measures similarity, or fuzziness, using a
+http://en.wikipedia.org/wiki/Levenshtein_distance[Levenshtein edit distance]. An
+edit distance is the number of one-character changes needed to turn one term
+into another. These changes can include:
 
-The `fuzzy` query generates matching terms that are within the
-maximum edit distance specified in `fuzziness` and then checks the term
-dictionary to find out which of those generated terms actually exist in the
-index. The final query uses up to `max_expansions` matching terms.
+* Changing a character (**b**ox → **f**ox)
+* Removing a character (**b**lack → lack)
+* Inserting a character (sic → sic**k**)
+* Transposing two adjacent characters (**ac**t → **ca**t)
 
-Here is a simple example:
+To find similar terms, the `fuzzy` query creates a set of all possible
+variations, or expansions, of the search term within a specified edit distance.
+The query then returns exact matches for each expansion.
 
-[source,js]
---------------------------------------------------
-GET /_search
-{
-    "query": {
-       "fuzzy" : { "user" : "ki" }
-    }
-}
---------------------------------------------------
-// CONSOLE
-
-Or with more advanced settings:
+[[fuzzy-query-ex-request]]
+==== Example request
```

Comment:
Kind of nice to have a short version example as well, as I imagine that what most users are using

Reply:
Re-added a simple example with 7b5e10aa887.